soroban-env-host/src/budget.rs

```diff
@@ -153,13 +167,13 @@ impl BudgetDimension {
     }
 
     pub fn is_over_budget(&self) -> bool {
+        if self.limit == u64::MAX {
+            return false;
+        }
```

Comment:
```suggestion
```

Reply:
This is redundant with the existing condition below.

Reply:
Thanks, fixed.